cypress/e2e/cloud/userProfile.multipleOrg.test.ts

```diff
@@ -1,60 +1,3 @@
-export const singleAccount = [
```

Comment:
Split this test file. Into `userProfile.multipleOrg` and `userProfile.multipleOrg`. No change in the test themselves.

Reply:
@wdoconnell  fyi

Reply:
I'm fine with this in principle - but shouldn't we do this in a separate PR? I'm happy to help out since I did a bunch of these tests earlier. I just don't think this belongs in the same PR as the iox updates, and chopping up the User Profile tests in the same PR as this IOX PR will mean we don't know if we're introducing test flake, since we're modifying more than 3 test files.

Reply:
As requested, breaking this PR into 2 parts.